src/classes/yaml-utils.ts

```diff
@@ -2,7 +2,10 @@ import fs from "fs";
 import * as path from "path";
 import YAML from "yaml";
 import OsUtils from "./os-utils";
-import { CommandNames } from "../interfaces/types";
+import { CommandNames, KubeArgs, YamlEnv } from "../interfaces/types";
+
+const { IP_ONE } = process.env;
+const { IP_TWO } = process.env;
```

Comment:
I'm assuming these will come from user input later?

Reply:
Creating a cluster is a matter of another issue, function createClusterYaml shouldn't actually be in this issue.